### PR TITLE
[Cleanup] Dropdowns order change

### PR DESCRIPTION
**Pull request by Civolilah**
Here are screenshots of the implemented solution to change the order in the dropdown and place the clone above the archive and restore:

Screenshot 2023-01-19 at 00 56 31

Screenshot 2023-01-19 at 00 30 17

Screenshot 2023-01-19 at 00 57 14

Screenshot 2023-01-19 at 01 02 20

@turbo124 I noticed four places with the wrong order so it was changed.

### CODE REVIEW

File: src/pages/purchase-orders/edit/components/Actions.tsx [hunk 1]
```diff
@@ -39,12 +39,14 @@ import {
...
 import { useQueryClient } from 'react-query';
 import { useNavigate } from 'react-router-dom';
+import { Divider } from 'components/cards/Divider';
 
 export type Action = (po: PurchaseOrder) => {
   label: string;
   onClick: () => unknown;
   hideIf?: boolean;
   icon?: ReactElement;
+  showDivider?: boolean;
```
Comment (beganovich) [inline src/pages/purchase-orders/edit/components/Actions.tsx:49]: Can we use the std actions array for this? Looks like we are doubling the interface.

https://github.com/invoiceninja/ui/blob/main/src/pages/quotes/common/hooks.tsx#L338

File: src/pages/purchase-orders/edit/components/Actions.tsx [hunk 2]
```diff
@@ -170,6 +172,7 @@ export function useActions() {
             .finally(() => invalidateCache(po.id)),
         hideIf: po.archived_at > 0,
         icon: <Icon element={MdArchive} />,
+        showDivider: true,
```
Comment (beganovich) [inline src/pages/purchase-orders/edit/components/Actions.tsx:175]: Please stick to this way of using dividers:

https://github.com/invoiceninja/ui/blob/main/src/pages/quotes/common/hooks.tsx#L338

File: src/pages/purchase-orders/edit/components/Actions.tsx [hunk 3]
```diff
@@ -217,13 +221,19 @@ export function Actions(props: Props) {
...
+              )}
+
+              <DropdownElement
+                key={index}
+                onClick={action(props.purchaseOrder).onClick}
+                icon={action(props.purchaseOrder).icon}
+              >
+                {action(props.purchaseOrder).label}
+              </DropdownElement>
+            </>
```
Comment (beganovich) [inline src/pages/purchase-orders/edit/components/Actions.tsx:236]: Can we refactor this using std `ResourceActions`?

https://github.com/invoiceninja/ui/blob/main/src/components/ResourceActions.tsx

**Review by beganovich — APPROVED** [review]
Looks good, thanks 👍

File: src/pages/purchase-orders/edit/components/Actions.tsx (hunk 1 above)
Comment (Civolilah) [reply src/pages/purchase-orders/edit/components/Actions.tsx]: Sure, the code has been refactored with the current convenction of handling more_actions.

File: src/pages/purchase-orders/edit/components/Actions.tsx (hunk 2 above)
Comment (Civolilah) [reply src/pages/purchase-orders/edit/components/Actions.tsx]: Sure, the code has been refactored with the current convenction of handling more_actions.

File: src/pages/purchase-orders/edit/components/Actions.tsx (hunk 3 above)
Comment (Civolilah) [reply src/pages/purchase-orders/edit/components/Actions.tsx]: Sure, the code has been refactored with the current convenction of handling more_actions.

**Review by beganovich — APPROVED** [review]
Looks good, thanks 👍

**Pull request merged**